scripts/deploy.sh

```
@@ -0,0 +1,79 @@
+MAX_RETRIES=10
+RETRY_INTERVAL=10
+
+for i in $(seq 1 $MAX_RETRIES); do
+  echo "Health check attempt ${i}/${MAX_RETRIES}..."
+
+  if curl -sf http://localhost:8080/actuator/health > /dev/null 2>&1; then
+    echo "Deployment successful!"
+    exit 0
+  fi
+
+  if [ $i -lt $MAX_RETRIES ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)